### ISSUE TITLE
Conditional meta tag in Head component throws an error when false

### ISSUE THREAD

**Issue by balu-lt** [inertiajs/inertia#1567]
### Version:

- `@inertiajs/vue3` version: 1.0.7
- `vue` version: 3.3.4

### Describe the problem:

I’m having trouble using conditional meta tag in my project.
Not sure exactly when it was broken, but it was working before.

Using `v-if` directive on meta tag in the `Head` component.
When the value is `false` or `null` it throws an error:

`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'toString')`

### Steps to reproduce:

```vue
<Head title="APP">
    <meta v-if="null" name="test" content="test" />
</Head>
```

**Comment by craigrileyuk** [comment]
Resolved in  #1570

Vue 3.3 changed its labelling of the vNodes which broke the previous patch. This one is now backwards-compatible once it's merged into main.

**Comment by reinink** [comment]
This fixed has been released as part of [v1.0.8](https://github.com/inertiajs/inertia/releases/tag/v1.0.8) 👍
